package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "@launchpadlab/lp-components",
-  "version": "7.0.2",
+  "version": "8.0.0",
```

Comment:
Let me know if you don't feel this is a major upgrade. I considered it one given the major upgrade of several of the libraries as well as the supported Node versions.